src/app/downloads/downloads-table.component.css

```diff
@@ -58,14 +58,19 @@ table.scroll-x th:first-child, table.scroll-x td:first-child {
 .table-navigation > button {
   text-align: left;
 }
-prx-checkbox, #episodeTitle button, #podcastTitle button {
+prx-checkbox {
   position: relative;
   margin-top: -4px;
   top: 4px;
   width: 100%;
   overflow: hidden;
   text-overflow: ellipsis;
 }
+.title {
```

Comment:
Fixes #174. The title text still isn't quite aligned with the other table columns, but let's worry about that when we can replace `<table>`